### ISSUE TITLE
Missing important parameter in OpenID configuration

### ISSUE THREAD

**Issue by miro-grapeup** [opensearch-project/documentation-website#96]
There is a quite well documented configuration settings for OpenID: https://opendistro.github.io/for-elasticsearch-docs/docs/security-configuration/openid-connect/ but it clearly misses the information that for openid the `openid_connect_idp` prefix is required:
https://github.com/opendistro-for-elasticsearch/security-advanced-modules/blob/e5d91955087c7365ac9a9ff33368521761976ee6/src/main/java/com/amazon/dlic/auth/http/jwt/keybyoidc/HTTPJwtKeyByOpenIdConnectAuthenticator.java#L57

So this is how it should be configured:
```
openid_auth_domain:
        http_enabled: true
        transport_enabled: true
        order: 1
        http_authenticator:
          type: openid
          challenge: false
          config:
            subject_key: preferred_username
            roles_key: roles
            openid_connect_idp:
              enable_ssl: true
              verify_hostnames: false
```

instead of:

```
openid_auth_domain:
        http_enabled: true
        transport_enabled: true
        order: 1
        http_authenticator:
          type: openid
          challenge: false
          config:
            enable_ssl: true
            verify_hostnames: false
            subject_key: preferred_username
            roles_key: roles
```

**Comment by andy840314** [comment]
There are more details here.
https://github.com/opensearch-project/security/issues/1056

**Comment by keithhc2** [comment]
Closed with #105
